client/catalog/src/data/client.js

```diff
@@ -1,31 +1,36 @@
 const CoreApi = {
-	log: {
-		scan: {},
-	},
 	datatype:
```

Comment:
todo us clean this up :)

what's the right way to do it? I've thought before it would be cool if we could somehow generate a typesafe client from Model. Obviously fine as is for now :)